package.json

```diff
@@ -23,7 +23,7 @@
   },
   "dependencies": {
     "bindings": "1.2.1",
-    "nan": "2.0.7"
+    "nan": "^2.0.7"
```

Comment:
@drewfish I have no faith in anyone. Mistakes happen. It is must more reliable to select a good working version and avoid issues where someone installed version X of module and I try to debug an issue but get version Y (patch, minor, or otherwise). This lets us specify the dependencies we have tested with and expect.

Additionally, I think it is a feature that we have to be explicit about dependency upgrades and cut a new release. This makes it clear when changes happened and downstream consumers are able to properly migrate or roll back without issues.

@hildjj There is no emergency here. I'm sure your code works on v5 and all this does is hold up a migration to v6 which is not an emergency given that it just came out days ago and that you can easily depend on a fork of this module if you need to for integration testing. Emergency is if we shipped a broken package under a patch release or something else critical breaking installs on versions of node that have been out for more than a day.